channeldb/payments.go

```diff
@@ -475,7 +475,7 @@ func serializePaymentCreationInfo(w io.Writer, c *PaymentCreationInfo) error {
 		return err
 	}
 
-	byteOrder.PutUint64(scratch[:], uint64(c.CreationDate.Unix()))
+	byteOrder.PutUint64(scratch[:], uint64(c.CreationTime.Unix()))
```

Comment:
this should be converted to unix nano as well

Reply:
Sgtm, but I will only change if @halseth approves. I have had to roll back too many non-essential improvements already.

Reply:
Added conversion to nano. Not in this commit though, but a later one.